tajo-core/src/main/resources/webapps/admin/header.jsp

```diff
@@ -26,6 +26,7 @@
     <li><a class='top_menu_item' href='query.jsp'>Query</a></li>
     <li><a class='top_menu_item' href='catalogview.jsp'>Catalog</a></li>
     <li><a class='top_menu_item' href='query_executor.jsp'>Execute Query</a></li>
+    <li><a class='top_menu_item' href='monitoring.jsp'>Basic Monitoring</a></li>
```

Comment:
Simply ```Monitoring``` looks more appropriate because we don't provide any complex or advanced monitoring.